lib/asana/version.rb

```diff
@@ -1,5 +1,5 @@
 #:nodoc:
 module Asana
   # Public: Version of the gem.
-  VERSION = '0.8.1'.freeze
+  VERSION = '0.8.2'.freeze
```

Comment:
This is a relatively chunky set of changes - should we increment the minor version to `0.9.0`?